doc/nrf/ug_zigbee_configuring_zboss_traces.rst

```diff
@@ -0,0 +1,158 @@
+.. _ug_zigbee_configuring_zboss_traces:
+
+Configuring ZBOSS traces in |NCS|
+#################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The :ref:`nrfxlib:zboss` (ZBOSS stack) comes included in the |NCS| in a set of precompiled libraries, which can complicate the debugging process.
+To help with that, the ZBOSS stack can be configured to print trace logs that allow you to trace the stack behavior.
+This page describes how to enable and configure ZBOSS trace logs.
+
+Trace logs are printed in binary form and require access to stack source files, which are to be decoded into log messages.
+Enabling trace logs can help with the debugging process even if you cannot decode them (due to lack of access to stack source files).''
+This is because of the information they can provide about the stack behavior whenever an issue is found or reproduced.
```

Comment:
We can just say that whenever end user finds any issue with ZBOSS stack, they can enable trace logs and collect them while reproducing issue. Then end user can provide log file to the Devzone team to help them with debugging process.

Reply:
I will look into this. Sounds like a good idea - just want to make sure it follows other NCS docs in terms of style and what is/is not allowed.

Reply:
I've seen other pages do this as well, so sounds like a good idea to me.